stable/artifactory/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
```

Comment:
Add empty line

Reply:
Hi, @eldada thanks a lot for the fast response. I have update artifactory-ha chart too and added an empty line as well. Cheers!